saleor/dashboard/order/forms.py

```diff
@@ -307,3 +310,40 @@ class OrderFilterForm(forms.Form):
 
 class PaymentFilterForm(forms.Form):
     status = forms.ChoiceField(choices=PAYMENT_STATUS_CHOICES)
+
+
+class AddDeliveryGroupItemForm(forms.Form):
+    variant = AjaxSelect2ChoiceField(queryset=ProductVariant.objects.all())
```

Comment:
I think we should allow to select only variants of available products (using function `get_available_products`).